k8s-test-runner/package.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Check if version argument is provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+# Capture the version from the script arguments
+VERSION=$1
+
+# Navigate to the cmd directory
+cd cmd/
+
+# Build the binary for Darwin ARM64 and append the version to its name
+GOOS=darwin GOARCH=arm64 go build -o k8s-test-runner-darwin-arm64
```

Comment:
It'd be cool if we could utilize `goreleaser` here, later https://github.com/smartcontractkit/chainlink/blob/develop/.goreleaser.develop.yaml
Provides a good speed boost and proper caching.

Reply:
@skudasov yeah, we can do it in the future if it becomes a priority